### PR TITLE
specification for RBDs list view

### PR DESCRIPTION
**Pull request by kamleshverma1**
specification for RBDs list view

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk @brainfunked @anivargi @shtripat @nthomas-redhat Please review it.

**Comment by kamleshverma1** [comment]
Addresses #75 .

File: specs/rbd_list_view_component.adoc [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+/api/1.0/GetClusterList - fetch the all detail along with RBDs list.
+
+Required from API - 
+
+* Response format for the API call
+* If "GetClusterList" contain RBDs list . than "parameter name" should be known which is  holding the RBDs list.
```
Comment (kamleshverma1) [inline specs/rbd_list_view_component.adoc]: @anivargi @shtripat @brainfunked If "GetClusterList" contain RBDs list . so Can i have  "parameter name" which is  holding the RBDs list ? or Is there any other APi for RBDs list ?

Comment (shtripat) [reply specs/rbd_list_view_component.adoc]: @kamleshverma1 I feel rbd listing should be a separate api like /clusters/{id}/GetRbdList. @anivargi can you confirm please?

Comment (mkudlej) [reply specs/rbd_list_view_component.adoc]: I agree with @shtripat RBD list should be separated API call.

Comment (anivargi) [reply specs/rbd_list_view_component.adoc]: @shtripat @mkudlej Currently the API responds with all cluster related objects in 1 API call like for ex: for Ceph we have pools, but we surely have individual API calls for all storage objects like pools, RBD's in the future.

@kamleshverma1 The info will be present in the GetClusterList API call, currently I am not aware of the structure of how the details are stored in etcd and what the attributes are called, so if @shtripat or @nthomas-redhat can have a instance which has the details populated then the API will automatically respond with the desired response.

**Comment by kamleshverma1** [comment]
@gnehapk @cloudbehl @anivargi @nthomas-redhat @shtripat Please review this spec . i have updated this.

File: specs/rbd_list_view_component.adoc [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+* Name
+
+* Cluster Name
+
+* backing pool
+
+Required from API -
+
+* status
```
Comment (kamleshverma1) [inline specs/rbd_list_view_component.adoc:62]: @anivargi @nthomas-redhat @shtripat according to UX mockup (https://redhat.invisionapp.com/share/BR8JDCGSQ#/screens/198417119) , missing these fields for RBDs in given response format by anup
1) status
2) utilization
3) alert count

Comment (shtripat) [reply specs/rbd_list_view_component.adoc:62]: @kamleshverma1 `utilization` and `alert count` would be done as part of monitoring I feel. I dont see a `status` field as such for rbds in cli output from ceph.

**Pull request closed without merging**